app/controllers/admin/students_controller.rb

```diff
@@ -1,60 +1,121 @@
 # frozen_string_literal: true
 
 module Admin
-  class StudentsController < ApplicationController
+  class StudentsController < Admin::LayoutController
     before_action :set_student, only: %i[show edit update destroy]
+    before_action :set_search, only: %i[index new create edit update]
+    include ExportableFormatConcern
 
     def index
-      @q = StudentAccount.ransack(params[:q])
       @students = @q.result(distinct: true).page(params[:page])
       @count = @students.total_count
-      @sort_fields = {
-        'Name': 'name asc',
-        'Email': 'email asc',
-        'Created At': 'created_at asc',
-        'Updated At': 'updated_at asc'
-      }
     end
 
     def show; end
 
     def new
-      @student = StudentAccount.new
+      @student_account = StudentAccount.new
+      @q = StudentAccount.ransack(params[:q])
     end
 
-    def edit; end
+    def edit
+      set_student
+    end
 
     def create
-      @student = StudentAccount.new(student_params)
-
-      if @student.save
-        redirect_to admin_students_path, notice: 'Student was successfully created.'
+      @student_account = StudentAccount.new(student_params)
+      if @student_account.save
+        redirect_to admin_students_path, notice: 'Student account created successfully.'
       else
+        errors_student_name(@student_account)
+        errors_student_email(@student_account)
+        errors_student_password(@student_account)
         render :new
       end
     end
 
     def update
```

Comment:
app/controllers/admin/students_controller.rb:37:5: C: Metrics/MethodLength: Method has too many lines. [12/10]


fix this rubocop issue